client/jetpack-cloud/sections/agency-dashboard/sites-overview/style.scss

```diff
@@ -136,7 +136,7 @@
 		rgba( 255, 255, 255, 1 ) 100%
 	);
 	top: 0;
-	right: 48px;
+	right: 75px;
 	@include break-large() {
 		right: 0;
 	}
```

Comment:
Better to replace `top` and `right` with RTL-aware rules like `inset-block-start` and `inset-inline-end` respectively.

Reply:
Thanks for the review, @manzoorwanijk. I have replaced all the `top` and `right` properties with `inset-block-start` & `inset-inline-end` respectively in this file.